config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
++aYyy3KOwChX5OciPJP0CefC4ecd7MoFLpSlQhwUeZ/hHCShz6APoJDGhwJX/e+Aj8Xd/4uyMd+6aQLYfcVNOLqLFY81nWVkV6B4kZSwC8cPGjlGWkNiMSbQ1iolCZIzOTMbIvnFBdYgXhyh0Ygc5g9jQnM5/7QHYoQyec6ZTFuYGEWg0qWBQOadmKXi4wLv3Y9c/F28DVmhidol31A08lvk08qk0Y9JCTqiDuZti1xr8G51DWRrapMmrK5jK9gbHmkQOxsCNvpLKPFRQYIV4wdaek+NHm7j/zXQ/z7wgFqVz27eYJCNsgZtR5x/gXJC2BoSONO4SOjT4STsEXZ1T+8NhjY1pvzUpGCMhNQ07ddl+WM5ZFUq7M3D4jh9be906WMnlvs1S1+HGHRogdg5hsDpS1aMWZQ2qmfz--unwcPXhMLqm4n4kR--j5H8d9XCeDnOqaEAgJ25+A==
```

Comment:
this should never be pushed to any repo as it contains ur local machine's encrypted secrets which will later differ from those on the server